.ci/deps.sh

```diff
@@ -22,6 +22,9 @@ deps_perl="perl libperl-critic-perl"
 deps_infer="m4 opam"
 sudo apt-get -y install $deps $deps_python_gi $deps_python_dbus $deps_perl $deps_infer
 
+# Chande environment for flawfinder from python to python2
+sudo sed -i'' '1s/.*/#!\/usr\/bin\/env python2/' /usr/bin/flawfinder
```

Comment:
ah right i does make any difference, i guess i should remove it then

2016-07-06 15:33 GMT+01:00 Zatreanu Adrian-Gabriel <notifications@github.com
>:

> In .ci/deps.sh
> <https://github.com/coala-analyzer/coala-bears/pull/566#discussion_r69741615>
> :
>
> > @@ -22,6 +22,9 @@ deps_perl="perl libperl-critic-perl"
> >  deps_infer="m4 opam"
> >  sudo apt-get -y install $deps $deps_python_gi $deps_python_dbus $deps_perl $deps_infer
> >
> > +# Chande environment for flawfinder from python to python2
> > +sudo sed -i'' '1s/.*/#!\/usr\/bin\/env python2/' /usr/bin/flawfinder
>
> what does the '' after -i do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/566/files/3b06ecc11e734f82fc6bb619063d8e3dc0b30368..e5cfe2cc18d130ee44f55c7d4c6b5a49303e8736#r69741615>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xotH8uKoAYc6tnngFGNZzi0QuTPP_ks5qS7yrgaJpZM4JF479>
> .
>